src/main/java/dgen/utils/schemas/Schema.java

```diff
@@ -0,0 +1,8 @@
+package dgen.utils.schemas;
+
+import dgen.utils.SpecificationException;
+
+public interface Schema {
```

Comment:
I've been using Schema and SchemaType to help differentiate these classes from the existing column and table classes.  I'm not entirely sure if there's a better name to describe it since they're mainly just used to load the specification into java objects